atomicapp/cli/main.py

```diff
@@ -53,9 +53,10 @@ def print_app_location(app_path):
 def cli_genanswers(args):
     try:
         argdict = args.__dict__
+        location = argdict['answers']
```

Comment:
I don't know if I like this. The reason you are converting from `answers` to `location` is because "answers" doesn't really make sense as a variable name for this I don't think.

How about we use `--destination` instead and the user could provide a path (`./path/to/myanswers.conf`) or a directory (`./path/to/`) in which the program will tack on the default filename to the directory. 

@jberkus, do you think `--answers` makes sense, or should we change it?